qemu/tests/fio_perf.py

```diff
@@ -95,38 +112,52 @@ def run(test, params, env):
     :param env: Dictionary with test environment
     """
 
-    def fio_thread(session, run_cmd, cmd_timeout):
+    def fio_thread():
         """
         run fio command in guest
-
-        :param session: VM session
-        :param run_cmd: fio command
-        :param cmd_timeout: Timeout in seconds
         """
         session.cmd_status(run_cmd, cmd_timeout)
```

Comment:
"session = vm.wait_for_login(timeout=login_timeout)" is a global parameter, so I can use it directly.